src/v2/api/controllers/TeacherController.ts

```diff
@@ -472,4 +472,24 @@ export class TeacherController {
   ) {
     return this.teacherService.connectTeacherWithCathedra(teacherId, cathedraId);
   }
+
+  @Delete('/:teacherId/cathedra/:cathedraId')
+  @ApiOkResponse({
+    type: TeacherWithRolesAndContactsResponse,
```

Comment:
Add description properties in ```TeacherWithRolesAndContactsResponse```

Reply:
I meant that you have to describe every property of this class
![image](https://github.com/fictadvisor/fictadvisor-api/assets/114052215/f4d18e9c-7e9d-46a9-9214-fbb53168b350)